packages/server/src/auth.ts

```diff
@@ -1,4 +1,4 @@
-import request from 'request';
+import axios from 'axios';
```

Comment:
What does axios do that we can't do with the node API?

Node 18 supports fetch: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API